stacks/firehose-stack.js

```diff
@@ -169,24 +169,37 @@ export function UcanFirehoseStack({ stack, app }) {
     databaseName,
     tableInput: {
       name: tableName,
-      partitionKeys: [],
+      partitionKeys: [
+        { name: 'day', type: 'date' },
+      ],
+      parameters: {
+        classification: "json",
+        typeOfData: "file",
+        "projection.enabled": "true",
+        "projection.day.type": "date",
+        "projection.day.format": "yyyy-MM-dd",
+        "projection.day.range": "2023-01-01,NOW",
```

Comment:
Can/should it be NOW minus `n` days/months?

Reply:
So I _think_ this is specifying the range that the query engine will use for the `day` parameter, so if we did that it would no longer query the `n` most recent days/months of data.

I think what you're looking for is the mechanism for limiting how much data gets scanned by the query engine - one of the coolest things about the way we've set this up is that all you need to do to do that is add a `WHERE` clause to a query that restricts the `day` parameter (ie, `WHERE day > (CURRENT_DATE - INTERVAL '2' DAY`) - Athena handles the rest!